CONTRIBUTING.md

```diff
@@ -231,6 +231,24 @@ Writing separate tests might be much better than making a single test that check
 
 Also, we should use driver methods as much as possible instead of testing commands directly via `RunCommand`.
 
+#### Integration tests naming guidelines
+
+1. Test files should be named after the command they test.
+   For example, `find_test.go` for testing the find command.
+   Test files belonging to the same commands group are organized into folders named after this group,
+   such as `aggregation`, `operations`, `diagnostic`, etc.
+2. If the file contains compatibility tests, add the `_compat` suffix. For example, `find_compat_test.go`.
+3. Test names should include the name of the command being tested.
+   For instance, `TestDistinct` for testing the distinct command.
+4. Compatibility tests should have `Compat` in the name, following the command.
+   For example, `TestDistinctCompat`.
+5. Test names should be descriptive and provide information about the functionality or condition being tested.
+   If the test is checking for a specific error scenario, include the error scenario in the name.
+6. Keep test names concise, avoiding overly cryptic names. Use abbreviations when appropriate.
+7. Avoid including test data in the name to maintain clarity and prevent excessively long names.
+8. Test case names should follow `TitleCase` capitalization style.
+9. Test and subtest names should not exceed 64 characters.
```

Comment:
In addition, 64 characters rule also adds `pg`, `sqlite` etc suffix, so it would be a bit shorter. :see_no_evil:

Reply:
I think this is about the test name _segments_